src/webots/nodes/WbCamera.cpp

```diff
@@ -1156,7 +1156,7 @@ void WbCamera::updateNoiseMaskUrl() {
     noiseMaskPath = completeUrl;
 
   const QString error = mWrenCamera->setNoiseMask(noiseMaskPath);
-  if (!error.isEmpty())
+  if (!completeUrl.isEmpty() && !error.isEmpty())
```

Comment:
Just wondering, in theory the `noiseMarkUrl` is also a `jpg` file and if that's the case, couldn't `completeUrl` value be `missingTexture()` too ?